tutorials/configure-enterprise-relay.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Configure your endpoints for Smallstep Enterprise Relay
+updated_at: December 08, 2025
+html_title: Configure your Apple endponts to use Smallstep's Enterprise MASQUE Relay
+description: This tutorial describes how to deploy Smallstep's enterprise MASQUE relay service
+---
+
+## Before you begin
+
+To get your Relay set up, you will need to give Smallstep the following information:
+
+- **Relay Trust Bundle**. This will be used by the Relay to verify client certificates.
+This bundle needs to include both Root and Intermediate CA certificates for any CAs you want your Relay to trust.
+A typical configuration will include your team's Smallstep Accounts Root and Intermediate CA.
+- **Relay Region**. The GCP region for the relay, eg. `US_CENTRAL1`
+
+## Client Configuration
+
+Once we have your details,
+Smallstep will create your relay server and give you the Relay URL,
+which you’ll need to configure clients.
+
+For most customers, the Relay will accept client certificates from your team's Smallstep Accounts CA.
+And, therefore, your clients will need to trust your team's Smallstep Accounts Root CA.
```

Comment:
I'm confused by this line. If the Relay accepts client certificates from the Smallstep Accounts CA, then why would clients need to be configured to trust the Smallstep Accounts CA? Clients would need to be configured to trust the Workloads CA, which is what you say in the paragraph below.